.circleci/config.yml

```diff
@@ -43,7 +47,7 @@ default-job: &default-job
     COREPACK_ENABLE_DOWNLOAD_PROMPT: '0'
   working_directory: /tmp/mui
   docker:
-    - image: cimg/node:20.19
+    - image: cimg/node:23.11
```

Comment:
Nitpick (clearly can be a follow up): It might make sense to sync the `@types/node` versions and the Netlify node version to maintain a unified set of API capabilities.